ingest/Ingest.py

```diff
@@ -0,0 +1,102 @@
+__author__ = 'Hao'
+
+import multiprocessing
+from SPARQLWrapper import SPARQLWrapper, JSON
+import itertools
+import json
+import requests
+
+class Ingest:
+    """say something"""
+
+    def __init__(self, object_index, object_type,
+                 get_objects_query_location, describe_object_query_location, variable_name_sparql):
+        self.object_index = object_index
+        self.object_type = object_type
+        self.records = []
+        self.get_objects_query = Ingest.load_file(get_objects_query_location)
+        self.describe_object_query = Ingest.load_file(describe_object_query_location)
+        self.variable_name_sparql = variable_name_sparql
+
+
+    def load_file(filepath):
```

Comment:
Let's start from this term: Indeed, I totally agree, using the term
"object" as a generalized substitution for different cases such as
dataset/publication/project/etc seems a bit ambiguous. What is a good term
that you suggest? "x"? "thing"? I suggest that we nail the name of this
term down first so that we don't have to change again.

On Sunday, November 1, 2015, Stephan Zednik <notifications@github.com>
wrote:

> In ingest/Ingest.py
> <https://github.com/tetherless-world/dco-elasticsearch/pull/29#discussion_r43588644>
> :
>
> > +import requests
> > +
> > +class Ingest:
> > +    """say something"""
> > +
> > +    def __init__(self, object_index, object_type,
> > +                 get_objects_query_location, describe_object_query_location, variable_name_sparql):
> > +        self.object_index = object_index
> > +        self.object_type = object_type
> > +        self.records = []
> > +        self.get_objects_query = Ingest.load_file(get_objects_query_location)
> > +        self.describe_object_query = Ingest.load_file(describe_object_query_location)
> > +        self.variable_name_sparql = variable_name_sparql
> > +
> > +
> > +    def load_file(filepath):
>
> Let's stop using the term object in the variable names and documentation
> as the term is very ambiguous.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tetherless-world/dco-elasticsearch/pull/29/files#r43588644>
> .
>